jmh-core/src/main/java/org/openjdk/jmh/runner/Runner.java

```diff
@@ -200,22 +203,13 @@ public RunResult runSingle() throws RunnerException {
 
             if (lock == null) {
                 String msg = "Unable to acquire the JMH lock (" + JMH_LOCK_FILE + "): already taken by another JMH instance";
-                if (JMH_LOCK_IGNORE) {
-                    out.println("# WARNING: " + msg + ", ignored by user's request.");
-                } else {
-                    throw new RunnerException("ERROR: " + msg + ", exiting. Use -Djmh.ignoreLock=true to forcefully continue.");
-                }
+                throw new RunnerException("ERROR: " + msg + ", exiting. Use -Djmh.ignoreLock=true to forcefully continue.");
```

Comment:
The local `msg` here was to use it in both branches. Since both branches are gone, this could be just: `throw new RunnerException("ERROR: Unable to acquire the JMH lock (" + JMH_LOCK_FILE + "), already taken by another JMH instance, exiting. Use -Djmh.ignoreLock=true to forcefully continue.");`